### PR TITLE
Add percentage difference in discrepancies check

### PR DESCRIPTION
**Pull request by liza-mazurova**

### CODE REVIEW

File: anyfin-platform/platform-composer/dags/cloudsql_to_bigquery/utils/etl_utils.py [hunk 1]
```diff
@@ -174,6 +174,11 @@ def fetch_bigquery_rowcount(self, ds, **kwargs):
             counts[row.table_name] = row.num_of_unique_rows
         return counts
 
+    # Return absolute number of percentage difference between two numbers
+    @staticmethod
+    def __calculate_percentage_difference(val1, val2):
+        print(val1, val2)
```
Comment (sofiathoren) [inline anyfin-platform/platform-composer/dags/cloudsql_to_bigquery/utils/etl_utils.py:180]: logging.info() maybe?

Comment (liza-mazurova) [reply anyfin-platform/platform-composer/dags/cloudsql_to_bigquery/utils/etl_utils.py:180]: oh, forget to delete

Comment (liza-mazurova) [reply anyfin-platform/platform-composer/dags/cloudsql_to_bigquery/utils/etl_utils.py]: fixed

**Review by sofiathoren — APPROVED** [review]
LGTM

**Review by csioa — APPROVED** [review]
LGTM

**Pull request merged**